dev/traffic_ops/cdn.json

```diff
@@ -10,6 +10,7 @@
 		"log_location_event": "stdout",
 		"whitelisted_oauth_urls": [],
 		"oauth_client_secret": "",
+		"oauth_user_attribute": "unique_name",
```

Comment:
Would these also need to be added to the ansible main.yml file?

Reply:
@zrhoffman do we need it there?

Reply:
>> Would these also need to be added to the ansible main.yml file?
>>
> @zrhoffman do we need it there?

The Traffic Ops Ansible role doesn't yet have OAuth stuff on the ATC master branch. @dsouza93 is adding it in [`dsouza93:lab_oath](https://github.com/dsouza93/trafficcontrol/commits/lab_oath) (but has not PRed it yet).

So, adding it in #7619 is not really possible without creating conflicts.